src/EnergyPlus/HeatBalanceManager.cc

```diff
@@ -530,13 +530,18 @@ namespace HeatBalanceManager {
 			NumObjects = GetNumObjectsFound( ConstrObjects( ONum ) );
 			for ( Loop = 1; Loop <= NumObjects; ++Loop ) {
 				GetObjectItem( ConstrObjects( ONum ), Loop, cAlphaArgs, NumAlphas, rNumericArgs, NumNumbers, Status );
-				if ( ONum != 5 ) {
-					CNum = FindItemInList( cAlphaArgs( 2 ), Construct );
-				} else {
+				if ( ONum == 5 ) {
 					CNum = FindItemInList( cAlphaArgs( 4 ), Construct );
+				} else {
+					CNum = FindItemInList( cAlphaArgs( 2 ), Construct );
 				}
 				if ( CNum == 0 ) continue;
 				Construct( CNum ).IsUsed = true;
+				if ( ONum == 4 || ONum == 6 ) {
+					// GroundHeatExchanger:Surface or EnergyManagementSystem:ConstructionIndexVariable
+					// Include all EMS constructions since they can potentially be used by a CTF surface
+					Construct( CNum ).IsUsedCTF = true;
```

Comment:
The integration test failure in EMSThermochromicWindow is coming from this I think.  The EMS construction actuator can work on windows as well as opaque.  By setting IsUsedCTF to true here, too much code runs in InitConductionTransferFunctions for window constructions and severe errors and fatal.  I suspect that the logic in this PR will have to be revised to retain use of TypeIsWindow along with the new IsUsedCTF.